cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -155,6 +155,120 @@ CREATE TABLE IF NOT EXISTS zipcodes (
 CREATE INDEX IF NOT EXISTS geom_index ON counties USING GIST (geom);
 CREATE UNIQUE INDEX IF NOT EXISTS zipcode_index ON zipcodes (zipcode);
 
+-- Precomputed table is required to ensure acceptable latency for the
+-- tileserver.
+CREATE TABLE IF NOT EXISTS demographics_by_state(
+    census_geo_id varchar(255) NOT NULL,
+    name varchar(255) NOT NULL,
+    black_population float,
```

Comment:
why is black / white population a different data type than total /under_five / poverty pop?

Reply:
Great question. Updated!